imports/plugins/included/ui-search/lib/components/searchModal.js

```diff
@@ -90,7 +146,80 @@ class SearchModal extends Component {
     );
   }
 
+  renderProductSearchFilter = () => {
+    const vendorOptions = this.props.vendors.map(vendor =>
+      <option key={vendor} value={vendor}>{vendor}</option>);
+
+    const tagsOptions = this.props.tags.map(tag =>
+      <option key={tag._id} value={tag._id}>{tag.name}</option>);
+
+    return (
+      <span className="rui product-filter pull-left">
+        <h3>Filter By</h3>
+        <div className="filter-options">
+          <span className="rui-options">
+        Vendors:
+            &nbsp;<select name="vendor" onChange={(event) => this.handleFilterChange(event)}>
+              <option value="">Select a Vendor</option>
+              {vendorOptions}
+            </select>
+          </span>
+          <span className="rui-options">
+        Price:
+          &nbsp;<select name="price" onChange={(event) => this.handleFilterChange(event)}>
+              <option value="">Select a price Range</option>
```

Comment:
Why do we have currency in USD?

Reply:
I could not find the Naira symbol on the keyboard do you think it would be cool to use an # instead? @saheedt

Reply:
Done @saheedt

Reply:
@Daramola98 it wouldn't be cool to use the # symbol, if you search 'naira symbol' on google, you will find it

Reply:
☝️